<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Please enter the Text segment binary file's full path:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to this MIPS Simulation Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add a memory file path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="memoryFile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
